internal/pkg/cli/deploy/env.go

```diff
@@ -383,6 +394,7 @@ func (d *envDeployer) buildStackInput(in *DeployEnvironmentInput) (*deploy.Creat
 		return nil, err
 	}
 	cidrPrefixListIDs, err := d.cidrPrefixLists(in)
+	publicALBSourceIPs := d.publicALBSourceIPList(in)
```

Comment:
This is splitting up the line above and line below, which should be adjacent. 

Can this just go below, in line 413 as
`PublicALBSourceIPs: d.publicALBSourceIPList(in)`?